docs/content/how-to/creating-tables.md

```diff
@@ -415,23 +425,16 @@ val dataset = spark.read.format("paimon").load("hdfs://path/to/table")
 
 {{< tab "Hive" >}}
 
-* To access existing paimon table, you can also register them as external and internal tables in Hive. The following SQL creates an external table named `my_table`, where the base path of table files is `hdfs://path/to/table`. As schemas are stored in table files, users do not need to write column definitions.
+To access existing paimon table, you can also register them as external tables in Hive. The following SQL creates an
+external table named `my_table`, where the base path of table files is `hdfs://path/to/table`. As schemas are stored
+in table files, users do not need to write column definitions.
 
 ```sql
-CREATE EXTERNAL TABLE my_table
-STORED BY 'org.apache.paimon.hive.PaimonStorageHandler'
-LOCATION 'hdfs://path/to/table';
-```
+SET hive.metastore.warehouse.dir=warehouse_path;
```

Comment:
No need to set `SET hive.metastore.warehouse.dir=warehouse_path` here.

Reply:
Good catch, fixed.